### PR TITLE
Add bidFloor to Yieldmo Adapter

### PR DESCRIPTION
**Pull request by HolzAndrew**
## Type of change

- [x] Feature

## Description of change
Add Bidfloor to Yieldmo Adapter. Also changes a local variable name.

@harquail @jasminenoack @elberdev @MelodyLi2017

### CODE REVIEW

File: modules/yieldmoBidAdapter.js [hunk 1]
```diff
@@ -103,8 +104,13 @@ function addPlacement(request) {
     callback_id: request.bidId,
     sizes: request.sizes
   }
-  if (request.params && request.params.placementId) {
-    placementInfo.ym_placement_id = request.params.placementId
+  if (request.params) {
+
```
Comment (MelodyLi2017) [inline modules/yieldmoBidAdapter.js]: ?

File: modules/yieldmoBidAdapter.js [hunk 2]
```diff
@@ -311,10 +317,10 @@ function isMraid() {
   return !!(window.mraid);
 }
 
-function getUserId(request) {
-  let userId;
-  if (request && request.userId && typeof request.userId === 'object') {
-    userId = request.userId;
+function getPubcId(request) {
+  let pubcid;
+  if (request && request.userId && request.userId.pubcid && typeof request.userId === 'object') {
```
Comment (MelodyLi2017) [inline modules/yieldmoBidAdapter.js]: do you need the typeof check?

File: modules/yieldmoBidAdapter.md [hunk 3]
```diff
@@ -23,7 +23,8 @@ var adUnits = [
        bids: [{
          bidder: 'yieldmo',
          params: {
-           placementId: '1779781193098233305' // string with at most 19 characters (may include numbers only) 
+           placementId: '1779781193098233305', // string with at most 19 characters (may include numbers only) 
+           bidFloor: .28 // optional param
```
Comment (MelodyLi2017) [inline modules/yieldmoBidAdapter.md]: string?

File: modules/yieldmoBidAdapter.js (hunk 1 above)
Comment (HolzAndrew) [reply modules/yieldmoBidAdapter.js]: derp. fixing.

File: modules/yieldmoBidAdapter.md (hunk 3 above)
Comment (HolzAndrew) [reply modules/yieldmoBidAdapter.md]: it should be a number

File: modules/yieldmoBidAdapter.js (hunk 2 above)
Comment (HolzAndrew) [reply modules/yieldmoBidAdapter.js]: NEED? no. extra security though

**Comment by bretg** [comment]
Needs a docs PR covering:
- addition of bidFloor param
- support for pubCommon ID -- add "userIds: pubCommon" to the jekyll variables on top

**Comment by HolzAndrew** [comment]
@bretg made [pr to docs](https://github.com/prebid/prebid.github.io/pull/1350/files)

**Comment by bretg** [comment]
Thanks - Docs PR https://github.com/prebid/prebid.github.io/pull/1350

**Pull request merged**